src/main/java/subway/line/Line.java

```diff
@@ -0,0 +1,85 @@
+package subway.line;
+
+import subway.station.Station;
+import subway.station.StationResponse;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Line {
+    private Long id;
+    private String name;
+    private String color;
+    private Long startStationId;
+    private Long endStationId;
+
+    public Line(Long id, String name, String color, Long startStationId, Long endStationId) {
+        this.id = id;
+        this.name = name;
+        this.color = color;
+        this.startStationId = startStationId;
+        this.endStationId = endStationId;
+    }
+
+    public Line(String name, String color, Long startStationId, Long endStationId) {
+        this(null, name, color, startStationId, endStationId);
+    }
+
+    public static Line fromRequest(LineRequest lineRequest) {
+        return new Line(lineRequest.getName(),
+                lineRequest.getColor(),
+                lineRequest.getUpStationId(),
+                lineRequest.getDownStationId());
+    }
```

Comment:
여기도 LineRequest를 받는 생성자를 오버라이딩 할수 있을 것 같아요.
```suggestion
public Line(LineRequest lineRequest) {
        this(lineRequest.getName(), lineRequest.getColor(), lineRequest.getUpStationId(), lineRequest.getDownStationId());
    }

Reply:
LineRequest 객체를 받아서 Line을 생성한다는 의미의 이름을 붙여주고 싶어서 정적 메서드로 만들어줬었는데요! 생성자 오버라이딩을 하는 게 더 직관적일까요??